### ISSUE TITLE
Add More OCaml to Books page

### ISSUE THREAD

**Issue by amirmc** [ocaml/v2.ocaml.org#554]
http://ocaml-book.com/more-ocaml-algorithms-methods-diversions/

**Comment by agarwal** [comment]
Related to #524.

**Comment by dbuenzli** [comment]
I'm currently going through the book and (somehow as expected) it's an excellent one. It's a shame it's still not mentioned on the website. @johnwhitington I don't think anyone will blame you if you take action to integrate it yourself.

**Comment by johnwhitington** [comment]
I've had a go at getting ocaml.org to build on my machine so I can begin to make contributions, but no luck yet (stuck on getting ocamlnet to build for now).

Perhaps the root of the problem is that something in "make deps" seems to rely on OCaml being exactly version 4.01.0, so everything I'm doing may be out of date. Can that requirement really still be needed?

```
=-=- Synchronising pinned packages =-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=  🐫 
[ocamlorg] /Users/john/repos/ocaml.org/ already up-to-date
[ERROR] ocamlorg is not available because your system doesn't comply with
        ocaml-version = "4.01.0".
```

**Comment by dbuenzli** [comment]
@johnwhitington You can keep this simple by simply using the pencil on the top right corner visible on any page of the website. This will lead you to github where you can simply edit the `.md` file by clicking again the pencil. I guess in this case you will maybe need to make a separate commit for adding the image file to the `img` directory.

**Comment by johnwhitington** [comment]
See Pull Request #657

**Comment by dsheets** [comment]
Fixed by #657.
